corehq/ex-submodules/soil/__init__.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, mimetype="text/plain",
         self.cache_backend = cache_backend
         # legacy default
         self.suffix = suffix or ''
+        self.message = message
```

Comment:
message is pretty generic, maybe add a comment here about what this parameter does